selfdrive/car/gm/carcontroller.py

```diff
@@ -14,6 +14,11 @@ def __init__(self, car_fingerprint):
       self.STEER_STEP = 2              # how often we update the steer cmd
       self.STEER_DELTA_UP = 7          # ~0.75s time to peak torque (255/50hz/0.75s)
       self.STEER_DELTA_DOWN = 17       # ~0.3s from peak torque to zero
+    elif car_fingerprint == CAR.ACADIA_DENALI:
```

Comment:
@CryptoKylan , why duplicating the code?
if car_fingerprint is in [CAR.VOLT, CAR.ACADIA_DENALI] should be ok, right?